pages/linux/telinit.md

```diff
@@ -0,0 +1,29 @@
+# telinit
+
+> Change SysV runlevel.
+> Since the concept SysV runlevels is obsolete the runlevel requests will be transparently translated into systemd unit activation requests.
+> More information: <https://manned.org/telinit>.
+
+- Power off the machine:
+
+`telinit 0`
+
+- Reboot the machine:
+
+`telinit 6`
+
+- Change SysV run level (2, 3, 4, 5):
+
+`telinit {{run_level_number}}`
```

Comment:
If these are all the possible values, then
```suggestion
- Change SysV run level:

`telinit {{2|3|4|5}}`
```

Reply:
Yeah, this seems much better.